### ISSUE TITLE
Don't see the file mentioned in ReadME

### ISSUE THREAD

**Issue by zuochen33** [ywangeq/instance_dataset_gen#2]
Hello, you said this project is your code to generate spherical images from raw lidar point clouds. But i can't find the code file . Am i missing something? Hope for your answer . Many thanks.

**Comment by lawrencekiba** [comment]
@zuochen33, I think it's not just you. I was looking around the repo for days & couldn't find the said code at read.me too. 

@ywangeq any idea as to when the code is going to be available?

**Comment by ywangeq** [comment]
@zuochen33 Sorry to make you feel trouble, due to the former issue which said this repo cannot work correctly.  So i guess that I may not implement it with the ipython and try to rewrite it with a useful main function interface. 
And @lawrencekiba @zuochen33  if you try to use it in a hurry, you can find the removed code (some weeks ago) with ipython version in the commit(rewrite for inference) track. All the functions are written in it to generate the spherical image.  
I will try my best to finish it before this month
Happy New Year~

**Comment by ywangeq** [comment]
@zuochen33 @lawrencekiba  I have uploaded the new version with the test. Please try it.

**Comment by lawrencekiba** [comment]
@ywangeq Appreciate the swift response. I have tried the latest version & it seems to work fine for now. Thank you for your work. Happy New Year.

**Comment by lawrencekiba** [comment]
> @ywangeq Appreciate the swift response. I have tried the latest version & it seems to work fine for now. Thank you for your work. Happy New Year.

Hi, I have some questions not related to this issue & have opened a new issue. Please take your time to reply when you return to work. #3

**Comment by zuochen33** [comment]
Thanks veeeery much!!!I'm new in automatic drive learning and this work helps me a lot ! I run this work  and generate .npy file. A new problem is that the output don't match the data released by SqueezeSeg, is there any way i can prove the data projection is correct？Sorry if i troubled you.
Thanks again.

**Comment by ywangeq** [comment]
@zuochen33  Can you explain more about the matching problem? Do you mean the shape or some other things?

**Comment by zuochen33** [comment]
@ywangeq The data shape are the same, but  the value are different. Especially, there are many zero value in the matrix. I think the output from this repo should be same as lidar_2d？

**Comment by ywangeq** [comment]
@zuochen33  No, they are not totally the same. I have explained the zero value in another issue which you can check it and understand the reason.
